src/day6.scala

```diff
@@ -0,0 +1,81 @@
+// using scala 3.1.0
+
+import scala.util.Using
+import scala.collection.mutable
+import scala.io.Source
+
+@main def part1(): Unit =
+  println(s"The solution is ${part1(readInput())}")
+
+@main def part2(): Unit =
+  println(s"The solution is ${part2(readInput())}")
+
+def readInput(): String =
+  Using.resource(Source.fromFile("input/day6"))(_.mkString.trim)
```

Comment:
It would be best to do the `trim` inside `parseSeveral`. The Scala.js UI in the webpage directly calls `part1(input)` rather than `part1()`, so if a visitor appends a newline in the input box, parsing will fail.

Reply:
Fixed in 46e4344